app/src/main/res/layout/app_version_fragment.xml

```diff
@@ -14,6 +14,7 @@
     android:layout_height="match_parent">
 
     <androidx.constraintlayout.widget.ConstraintLayout
+      android:id="@+id/app_version_fragment"
```

Comment:
Rename to `app_version_container` and `profile_list_container`

Reply:
Renaming done from `app_version_fragment` to `app_version_container` but `profile_list_container `was already assigned to `profile_list_activity` so I used that only and removed id(s) from the `profile_list_fragment`: https://github.com/oppia/oppia-android/pull/3185/commits/586a89a87edeffeb1c2f2bedbe1f2c19ec255774